README.md

```diff
@@ -41,12 +41,13 @@ Kotlin), [enable, upgrade, or install that plugin](https://www.jetbrains.com/hel
  [refresh the Gradle projects](https://www.jetbrains.com/help/idea/jetgradle-tool-window.html). 
 
 ### IntelliJ Build Configuration
-JDK version 1.8 is required for building and developing for IntelliJ IDEA Community
-Edition.
-1. Using IntelliJ IDEA, [configure](https://www.jetbrains.com/help/idea/sdk.html) a JSDK
-   named "**1.8**", pointing to `<JDK_18_HOME>`.
-   * If not already present, add `<JDK_18_HOME>/lib/tools.jar` [to the Classpath](https://www.jetbrains.com/help/idea/sdk.html#manage_sdks) tab for the **1.8** JSDK.
-2. Also configure a JSDK named "**IDEA jdk**" (case sensitive), pointing to `<JDK_16_HOME>`. If you don’t want to install JDK 1.6 then you may configure **IDEA jdk** to point to `<JDK_18_HOME>`. However, you must be careful to avoid using Java 8 APIs in IntelliJ IDEA Community Edition modules that use **IDEA jdk**. 
+JDK version 1.8 is required for building and developing for IntelliJ IDEA Community Edition.
+1. Using IntelliJ IDEA, [configure](https://www.jetbrains.com/help/idea/sdk.html) a JSDK named "**1.8**", pointing to `<JDK_18_HOME>`.
+   * If not already present, add `<JDK_18_HOME>/lib/tools.jar` [to the Classpath](https://www.jetbrains.com/help/idea/sdk.html#manage_sdks) tab for the **1.8** JSDK.  
+2. Configure a JSDK named "**IDEA jdk**" (case sensitive) pointing to `<JDK_18_HOME>`. This is the recommended build configuration if you are developing 
+   new plugin projects using the IntelliJ Platform SDK.  
```

Comment:
If you're developing a plugin project there is no need to configure 'IDEA jdk' or '1.8' JDK at all. You either need to use gradle-intellij-plugin, or setup an 'IntelliJ Platform Plugin SDK'.